--- graphviz/backend.py
+++ graphviz/backend.py
@@ -121,8 +121,12 @@
     if formatter is not None and formatter not in FORMATTERS:
         raise ValueError('unknown formatter: %r' % formatter)
 
+    if PLATFORM == 'windows':
+        engine = '%s.bat' % engine
     output_format = [f for f in (format_, renderer, formatter) if f is not None]
     cmd = ['dot', '-K%s' % engine, '-T%s' % ':'.join(output_format)]
+    if PLATFORM == 'windows':
+        cmd[0] = 'dot.bat'
 
     if filepath is None:
         rendered = None
